chromium_src/third_party/blink/common/features.cc

```diff
@@ -74,5 +74,10 @@ BASE_FEATURE(kBraveRoundTimeStamps,
              "BraveRoundTimeStamps",
              base::FEATURE_ENABLED_BY_DEFAULT);
 
+// Enabled HTTPS by Default.
+BASE_FEATURE(kHttpsByDefault,
```

Comment:
I don't have a strong feeling either way. Because kBraveTorWindowsHttpsOnly is in this file and it's closely related, I will leave it here for now, but I'm happy to move them both in another bug if we decide to.